app/views/projects/tasks/_summary.html.erb

```diff
@@ -0,0 +1,38 @@
+<section class="summary-panel">
+  <h4 class="header-underline">My Tasks
+    <span class="actions pt-1">
+      <div class="action">
+        <%= link_to project_tasks_path(current_project) do %>
+          <i class="fa-solid fa-tasks sidebar-link-icon"></i> All Tasks
+        <% end %>
+      </div>
+    </span>
+  </h4>
+  <% if @tasks.any? %>
+    <ul class="list-group tasks">
+      <% @tasks.first(@tasks_limit).each do |task| %>
```

Comment:
How about limiting this list in the controller instead and having `TASKS_LIMIT` as a constant